### ISSUE TITLE
Custome theme color

### ISSUE THREAD

**Issue by Shadowspear123** [dslul/openboard#124]
Thanks for the great app! Is it possible to add a custom background color option, besides the built in 4 themes? It would allow for a lot more customization. Simple keyboard (https://github.com/rkkr/simple-keyboard) has this option.

Thanks!

**Comment by awais1281** [comment]
yes, It's possible. you can add as many custom layouts you want. You can add custom color background as well as image background.

**Comment by szaimen** [comment]
> yes, It's possible. you can add as many custom layouts you want. You can add custom color background as well as image background.

Where can I do this in Openboard?

**Comment by Shadowspear123** [comment]
> yes, It's possible. you can add as many custom layouts you want. You can add custom color background as well as image background.

I don't think this is currently available in Openboard.

**Comment by awais1281** [comment]
> > yes, It's possible. you can add as many custom layouts you want. You can add custom color background as well as image background.
> 
> I don't think this is currently available in Openboard. 

Yes, It's not currently available but you can add more themes. If you are referencing to color picker like in simple keyboard, look at this commit you can include this one in open-board too. [https://github.com/rkkr/simple-keyboard/commit/797e3b5d5ccb910c6f579672e4647c3bf9bf0167](url)

**Comment by Fzndox** [comment]
Just using the material theme to make an AMOLED black would be great.

**Comment by dslul** [comment]
> Just using the material theme to make an AMOLED black would be great.

Well, that's what I did in the latest release! But it's taking a lot of time to be approved by Google...

**Comment by Fzndox** [comment]
> 
> 
> > Just using the material theme to make an AMOLED black would be great.
> 
> Well, that's what I did in the latest release! But it's taking a lot of time to be approved by Google...

Great news, I've been downloading from fdroid but im downloading the 1.4apk from your releases as we speak. TYSM!

**Comment by dslul** [comment]
> > > Just using the material theme to make an AMOLED black would be great.
> > 
> > 
> > Well, that's what I did in the latest release! But it's taking a lot of time to be approved by Google...
> 
> Great news, I've been downloading from fdroid but im downloading the 1.4apk from your releases as we speak. TYSM!

be aware that if you want to return to the F-Droid version (and also from F-Droid to mine), the signing key is different and you will lose your predictions history!

**Comment by alekksander** [comment]
yes please. simple color picker/predefined from colorful pallete could already be sufficent.
